README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is thelevel of imrtance in which the code is read. Generally helps the machine read the code and know what in what order it needs to follow it. what is it and order to listen to it first
+
```

Comment:
It also helps clarify the code for you and any programers that come afterwards. It also helps with Indexing and things like SEO